tests/cli/test_cli.py

```diff
@@ -21,6 +27,34 @@ def generate_test_input_file():
     return file_path
 
 
+def assert_equal_lists(res, expected):
```

Comment:
Hmm, my understanding is that

```
assert res == expected
```

will work fine for lists. If you're concerned about arbitrary types of iterable, you can do

```
assert list(res) == list(expected)
```